algoliasearch/src/main/java/com/algolia/search/saas/Query.java

```diff
@@ -714,12 +737,69 @@ public int hashCode() {
                                 Double.parseDouble(fields[2 * i + 1])
```

Comment:
This call can throw a `NumberFormatException` (unchecked exception), therefore `parse(...)` can throw, therefore `getInsidePolygon()` can throw... which is probably not a good idea. :/